src/common/components/issue-filing-button.tsx

```diff
@@ -49,6 +54,17 @@ export class IssueFilingButton extends React.Component<IssueFilingButtonProps, I
             : null;
         const target: string = isSettingValid ? '_blank' : '_self';
 
+        const needsSettingsContentProps: IssueFilinigNeedsSettingsContentProps = {
```

Comment:
`IssueFilinigNeedsSettingsContentProps` only has property `isOpen` but there are lots of other properties below; this seems strange.